src/ui-config/reservePatches.ts

```diff
@@ -103,11 +103,24 @@ export function fetchIconSymbolAndName({ underlyingAsset, symbol, name }: IconSy
       symbol: 'KNCL',
       iconSymbol: 'KNCL',
     },
+    // arbitrum
     '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8': {
       name: 'Bridged USDC',
       symbol: 'USDC.e',
       iconSymbol: 'USDC',
     },
+    // optimism
+    '0x7f5c764cbc14f9669b88837ca1490cca17c31607': {
```

Comment:
@grothem are these in address book?

Reply:
oh yeah good call, I'll update to pull from there

Reply:
Ok fixed the USDC ones, we could also update the other addresses in here too, but I didn't want to introduce a bunch of other changes with this PR